src/V3Hasher.cpp

```diff
@@ -465,10 +465,12 @@ class HasherVisitor final : public VNVisitorConst {
         });
     }
     void visit(AstCellInline* nodep) override {
-        m_hash += hashNodeAndIterate(nodep, HASH_DTYPE, HASH_CHILDREN, [this, nodep]() {
-            m_hash += nodep->name();
-            iterateConstNull(nodep->scopep());
-        });
+        m_hash += hashNodeAndIterate(nodep, HASH_DTYPE, HASH_CHILDREN,
+                                     [this, nodep]() { m_hash += nodep->name(); });
```

Comment:
This is relatively expensive, why is it required?

Reply:
Oh, sorry I missed name was compared before, this is fine then.

Reply:
nw, formatting really did a number on this